go/client/cmd_wot_list.go

```diff
@@ -29,28 +29,38 @@ func newCmdWotList(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comma
 		Action: func(c *cli.Context) {
 			cl.ChooseCommand(cmd, "list", c)
 		},
-		Flags: []cli.Flag{},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "voucher",
+				Usage: "Only get vouches written by me",
+			},
+		},
 	}
 }
 
 func (c *cmdWotList) ParseArgv(ctx *cli.Context) error {
 	if len(ctx.Args()) > 1 {
 		return errors.New("too many arguments")
 	}
-	c.ownWot = true
 	if len(ctx.Args()) == 1 {
 		username := ctx.Args()[0]
-		c.username = &username
-		c.ownWot = false
+		c.vouchee = &username
+	}
+	voucher := ctx.String("by-me")
+	if len(voucher) > 0 {
+		c.voucher = &voucher
```

Comment:
this doesn't look right.

Reply:
can confirm. pulled this down. tested locally.
`kbu wot list --voucher` is not showing a proposed vouch. did you test this? is it working for you?

Reply:
did you pass in a username for the voucher flag?

i updated this anyway; try again? this is an example of what i see, when requesting all vouches written by bob:
```
> ./keybase wot list --voucher bob
▶ WARNING Running in devel mode
Web-Of-Trust by bob
-------------------------------
Voucher: bob
Attestation: "by bob 1"
Status: PROPOSED
    `keybase wot revoke bob` to revoke your proposed vouch (coming soon)
Additional Details: {UsernameVerifiedVia:in_person Proofs:[] Other:}
-------------------------------
```

Reply:
i did a fresh pull and now i'm seeing the right vouches but there are still some issues...
```
Web-Of-Trust by luncher8
-------------------------------
Voucher: luncher8
Attestation: "luncher8 think luncher7 lunches"
Status: PROPOSED
    `keybase wot revoke luncher8` to revoke your proposed vouch (coming soon)
Additional Details: {UsernameVerifiedVia:audio Proofs:[] Other:}
-------------------------------
Voucher: luncher8
Attestation: "luncher4 is great"
Status: PROPOSED
    `keybase wot revoke luncher8` to revoke your proposed vouch (coming soon)
Additional Details: {UsernameVerifiedVia:in_person Proofs:[] Other:}
-------------------------------
```

1. when we're listing by voucher, we should probably display the `Vouchee` also, no? if i hadn't included the name of the vouchee in each of these vouches, there'd be no way to know who it was for.
2. it seems like vouchee is actually calculated incorrectly here. `keybase wot revoke luncher8` would be revoking by the voucher not the vouchee. that sort of fixes number 1. but seeing this now, i'd suggest we show both the `Voucher` and the `Vouchee` always. 
3. it's weird to specify `voucher` and have to pass in your own username. and it's functionality we really don't need. i thought `--voucher` would be a flag not a parameter (like `--by-me`) and the description is pretty misleading for this. my suggestion is to remove all of the custom voucher setting behavior, and change this to a boolean flag. if it's there, we show everything where the current user is the voucher. if there's a username, we show that user's `accepted` web of trust. and the CLI can just error if you try to pass both. two paths. that's all we really need.

Reply:
@mmou ^ these are all still issues

Reply:
the commit i just pushed should address these points